db/migrate/20230328181907_add_index_to_comments.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndexToComments < ActiveRecord::Migration[7.0]
+  def change
+  add_foreign_key :comments, :users, column: :user_id
```

Comment:
- [ ] Please kindly ensure that the foreign  key references the `author_id` as specified in the schema diagram